README.md

```diff
@@ -7,6 +7,33 @@ The IBM Blockchain extension allows a user to write and test chaincode projects
 * Generate skeleton chaincode projects
 * Adds a Fabric view to connect and interact with Fabric Blockchain networks
 
+
+## Connecting to a custom Hyperledger Fabric instance
```

Comment:
I'm not sure `custom` is the right word - could it not just be `Connecting to your own Hyperledger Fabric instance`?